panacea/lib/panacea/asclepius/remote/http.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Panacea.Asclepius.Remote.HTTP do
+  @behaviour Panacea.Asclepius.Remote
+
+  @asclepius_uri Keyword.get(Application.get_env(:panacea, :asclepius), :uri)
+  @ping_timeout 50
+  # one minute timeout for now
+  @ddi_timeout 60 * 1000
+
+  def ping do
+    asclepius_uri("/ping")
+    |> HTTPoison.get([recv_timeout: @ping_timeout])
+    |> case() do
```

Comment:
Might change it to pipe into `decode_response` instead.
Then all of the `Remote` callbacks will have the same return type -> `{:ok, response} | {:error, reason}`.
Callers can then choose to ignore or use the response/error